code/style.css

```diff
@@ -1,7 +1,254 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+@import url('https://fonts.googleapis.com/css2?family=Major+Mono+Display&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Space+Mono:wght@700&display=swap');
+
+body {
+  margin: 0; 
+}
+
+  .header-wrapper {
+    margin: 3px; 
+    width: 100vw;
+    height: 15vh; 
+    background-color:#000;
+    color: #fff;
+    display: flex;
+    justify-content: space-between;
+    font-family: 'Major Mono Display', monospace;
+    font-weight: 900;
+}
+
+  .pap {
+    margin-top: 25px; 
+    margin-left: 10px; 
+    margin-bottom: 10px; 
+    font-size: 40px;
+}
+
+  .header-r {
+    position: relative;
+    margin: 3px;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .hamburger {
+    position: relative;
+    margin-top:15px;
+    margin-right: 0;
+    margin-bottom: 0;
+    margin-left: 0; 
+    font-size: 60px;
+    cursor: progress;
+  }
+
+  .contact {
+    margin-right: 10px;
+    border-radius: 30%;
+		margin: 2px 2px;
+		padding: 2px 2px;
```

Comment:
uups, let's fix those indents.
Moreover, since both horizontal and vertical values are the same, we can simplify them :

```
margin: 2px;
padding: 2px;
```